velox/docs/develop/operators.rst

```diff
@@ -224,12 +224,12 @@ and emitting results.
      - A list of output columns. This is a subset of columns available in the left and right inputs of the join. The columns may appear in different order than in the input.
 
 NestedLoopJoinNode
-~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~
```

Comment:
Unintended change? CC: @majetideepak

Reply:
Sorry, I may have done it incorrectly during the process of rebasing.

Reply:
No problem. Thank you for fixing.